docs/user/exercises/programming-repository-access.inc

```diff
@@ -0,0 +1,130 @@
+The following tables lists the different types of repositories and gives an overview of the access rights different users have.
+To gain these access rights, a user must assume the indicated role in the course the repository belongs to.
+
+The different repository types are:
+
+  - **Base**: This includes all repositories that are set up when the exercise is created (template repository, solution repository, tests repository, auxiliary repositories).
+  - **Student Assignment**: A student's assignment repository copied from the template repository. This includes team assignment repositories.
+  - **Teaching Assistant (TA) Assignment**: An assignment repository created by a teaching assistant for themself.
+  - **Instructor Assignment**: An assignment repository created by an editor or instructor for themselves. Not available for exam exercises.
+  - **Student Practice**: A student's practice repository copied either from the template repository, or from their assignment repository. Can only be created after the due date of the exercise has passed. Not available for exam exercises.
+  - **Teaching Assistant (TA) Practice**: A practice repository created by a teaching assistant for themself. Not available for exam exercises.
+  - **Instructor Practice**: A practice repository created by an editor or instructor for themselves. Not available for exam exercises.
+  - **Instructor Exam Test Run**: A test run repository created by an editor or instructor for an exam. An instructor can create an exam test run before the start date of the exam to allow the instructor to test the exam from a student perspective before releasing it. This repository should be deleted before the exam is conducted.
```

Comment:
```suggestion
:Base: This includes all repositories that are set up when the exercise is created (template repository, solution repository, tests repository, auxiliary repositories).
:Student Assignment: A student's assignment repository copied from the template repository. This includes team assignment repositories.
:Teaching Assistant (TA) Assignment: An assignment repository created by a teaching assistant for themself.
:Instructor Assignment: An assignment repository created by an editor or instructor for themselves. Not available for exam exercises.
:Student Practice: A student's practice repository copied either from the template repository, or from their assignment repository. Can only be created after the due date of the exercise has passed. Not available for exam exercises.
:Teaching Assistant (TA) Practice: A practice repository created by a teaching assistant for themself. Not available for exam exercises.
:Instructor Practice: A practice repository created by an editor or instructor for themselves. Not available for exam exercises.
:Instructor Exam Test Run: A test run repository created by an editor or instructor for an exam. An instructor can create an exam test run before the start date of the exam to allow the instructor to test the exam from a student perspective before releasing it. This repository should be deleted before the exam is conducted.
```
rst supports description lists. Those might be better suited for this and the ones below.

![grafik](https://user-images.githubusercontent.com/64250573/230740953-b9e092f4-3049-49f9-8d4a-66a53ca688b8.png)